### PR TITLE
change defaults

### PR DESCRIPTION
**Pull request by sumo-drosiek**
- Use sumologic authenticator by default
- Use otlp formats by default

### CODE REVIEW

File: pkg/exporter/sumologicexporter/README.md [hunk 1]
```diff
@@ -59,6 +59,13 @@ exporters:
       - <regex1>
       - <regex2>
 
+    # instructs sumologicextension to use automatically generated sumologic endpoint
+    # to use direct endpoint, set it to null
+    # see sumologicextension documentation for details
+    # default = "sumologic"
+    auth:
+      authenticator: {"sumologic", null}
```
Comment (pmalek-sumo) [inline pkg/exporter/sumologicexporter/README.md:67]:
```suggestion
    # instructs sumologicextension to use automatically generated sumologic endpoint;
    # to use direct endpoint, set it to null
    # see sumologicextension documentation for details
    # default = "sumologic"
    auth:
      authenticator: {"sumologic", null}
```

**Review by pmalek-sumo — APPROVED** [review]

File: pkg/exporter/sumologicexporter/README.md [hunk 2]
```diff
@@ -59,6 +59,13 @@ exporters:
       - <regex1>
       - <regex2>
 
+    # instructs sumologicextension to use automatically generated sumologic endpoint;
+    # to use direct endpoint, set it to null
+    # see sumologicextension documentation for details
+    # default = "sumologic"
+    auth:
+      authenticator: {"sumologic", null}
```
Comment (pmalek-sumo) [inline pkg/exporter/sumologicexporter/README.md:67]: Actually can we change to to not include quotation marks as other defaults in comments here? They are not required for yaml so let's not add complexity where not needed.

Comment (sumo-drosiek) [reply pkg/exporter/sumologicexporter/README.md:67]: meh, auto merge :disappointed:

Comment (sumo-drosiek) [reply pkg/exporter/sumologicexporter/README.md:67]: https://github.com/SumoLogic/sumologic-otel-collector/pull/85

**Pull request merged**